src/functions-reference/probability_function_conventions.Rmd

```diff
@@ -1,4 +1,4 @@
-# Conventions for Probability Functions
+# Probability Function Conventions
```

Comment:
please don't change chapter names - it messes up how we do the linking into latest version.
you should also change the filename back to how it was, since filename matches chapter name.

Reply:
Oh okay, can change this back. Does that mean the new way of doing the `_lupdf` s is okay?